docs/sync.js

```diff
@@ -163,6 +163,7 @@ class Sync {
      * @param {string} filterRegex - A regular expression used to determine which changed Realms should trigger events. Use `.*` to match all Realms.
      * @param {string} name - The name of the event.
      * @param {function(changeEvent)} changeCallback - The callback to invoke with the events.
+     * @return {Promise<void>} A promise which is resolved when the worker has started.
```

Comment:
@returns ?

Reply:
I think both are allowed.

Reply:
oh if so proceed at will. maybe for consistency. but no big deal